### ISSUE TITLE
The Reference Panel would like to see journal paper acknowledgement guidelines in the user interface

### ISSUE THREAD

**Issue by hol353** [APSIMInitiative/ApsimX#3813]
Some text shown to the user in the user interface along the lines of:

The APSIM Initiative requests to use the following reference when citing APSIM Next Generation:
Holzworth, D., Huth, N.I., Fainges, J., Brown, H., Zurcher, E., Cichota, R., Verrall, S., Herrmann, N.I., Zheng, B., Snow, V., 2018. APSIM Next Generation: Overcoming challenges in modernising a farming systems model. Environmental Modelling & Software 103, 43–51. https://doi.org/10.1016/j.envsoft.2018.02.002

In addition, the APSIM Initiative recommends the following acknowledgement be added to all journal papers that use APSIM.
"Acknowledgment is made to the APSIM Initiative who manages and maintains the APSIM software which is provided free for research and development use.  The APSIM Initiative provides for distribution of modifications, promotes central control for quality assurance purposes, and considers structured innovation relevant to APSIM and its users."

**Comment by hol353** [comment]
@APSIMInitiative/reference-panel. FYI for your comment and suggestions.

**Comment by peter-devoil** [comment]
I'm not sure the last sentence is required - an acknowledgement is usually only a single sentence in itself.

**Comment by byzheng** [comment]
How about a footnote for the last sentence?

**Comment by yashvirchauhan** [comment]
It will be good to also give an example of APSIM classic 
The last para of acknowledgement looks a bit complicated. I recently acknowledged as given below:

"The authors also would like to acknowledge the APSIM initiative for providing APSIM software used in this study."

**Comment by sno036** [comment]
How about 

Acknowledgment is made to the APSIM Initiative which manages quality assurance and a structured innovation/improvement programme for the APSIM software which is then provided free for research and development use. The APSIM Initiative welcomes the user community to contribute their own improvements to that all users can benefit. See www.apsim.info for details.

For Classic just substitute in the Holzworth et al 2014 citation.

**Comment by sarahcleary** [comment]
Thanks Val.  I think that works well and happy to suggest this.  I note the comments that only one sentence is usually used.  The first sentence stands alone well.  I propose we suggest the above paragraph - it's only a suggestion/request anyway.  Thoughts?

**Comment by hol353** [comment]
Val's suggestion looks good to me as well.

**Comment by peter-devoil** [comment]
I think that https://github.com/APSIMInitiative/ApsimX/issues/3813#issuecomment-490705259 has been accepted by the RP. I'll put it into classic.

**Comment by kchenu** [comment]
I like Val suggestion (https://github.com/APSIMInitiative/ApsimX/issues/3813#issuecomment-490705259) but would suggest keep it to one sentence, and refering to the APSIM **model** rather than the APSIM software

> Acknowledgment is made to the APSIM Initiative, which manages quality assurance and a structured innovation programme for the APSIM model that is provided free for research and development use (see www.apsim.info for details).

**Comment by yashvirchauhan** [comment]
Agree what Karine just proposed and also make it clear that it is a software product, could I modify it a bit more:
"Acknowledgment is made to the APSIM Initiative responsible for quality assurance and a structured innovation programme for the APSIM model software, which is provided free for research and development use (see www.apsim.info for details)."

**Comment by hol353** [comment]
I would much prefer **APSIM Software** rather than **APSIM model software**. To me, APSIM isn't a model. It is a group (or framework) of models.

**Comment by kchenu** [comment]
What about:

"Acknowledgment is made to the APSIM Initiative responsible for quality assurance and a structured innovation programme for the Agricultural Production Systems sIMulator (APSIM), which is provided free for research and development use (see www.apsim.info for details)."

or just 
"Acknowledgment is made to the APSIM Initiative responsible for quality assurance and a structured innovation programme for APSIM, which is provided free for research and development use (see www.apsim.info for details)."

**Comment by hol353** [comment]
Yep this sounds good. I'd probably go with number two. No need to spell out APSIM in the acknowledgements section of a paper. It would already have been done by that point.

**Comment by sno036** [comment]
Just tweaking the language 

"Acknowledgment is made to the APSIM Initiative which takes responsibility for quality assurance and a structured innovation programme for APSIM's software, which is provided free for research and development use (see www.apsim.info for details)."

**Comment by kchenu** [comment]
Honestly, I don't like the term software here.

**Comment by sarahcleary** [comment]
@kchenu - any suggested change?  would "modelling software" be OK?  or platform?

"Acknowledgment is made to the APSIM Initiative which takes responsibility for quality assurance and a structured innovation programme for APSIM's modelling software, which is provided free for research and development use (see www.apsim.info for details)."

**Comment by sarahcleary** [comment]
How about this:

Acknowledgment is made to the APSIM Initiative which takes responsibility for quality assurance and a structured innovation programme for APSIM. APSIM is provided free for research and development use (see www.apsim.info for details).

**Comment by kchenu** [comment]
I'm happy with this

**Comment by sarahcleary** [comment]
Website updated:
http://www.apsim.info/AboutUs/APSIMModel.aspx
http://www.apsim.info/WelcometoAPSIM/tabid/55/ArtMID/592/ArticleID/1081/Default.aspx

@hol353 - can you please add to the user interface?

**Comment by hol353** [comment]
I have added it to the APSIM Next Generation user interface. We're working on APSIM 7.10 next.

**Comment by hol430** [comment]
Even if you merge the change into apsim classic, the text still won't appear in version 7.10 (the version which people download). The only people who will see the new text will be the 3-4 people who build apsim classic from source.

**Comment by hol353** [comment]
True but we are moving APSIM 7.10 to a continuous release system similar to Next Gen.

**Comment by hol430** [comment]
This text is now shown after clicking the help button in the UI. Can this issue be closed?
